crypto/keys/src/ristretto.rs

```diff
@@ -52,6 +52,14 @@ use zeroize::Zeroize;
 #[derive(Clone, Copy, Default, Zeroize)]
 pub struct RistrettoPrivate(pub(crate) Scalar);
 
+impl Eq for RistrettoPrivate {}
```

Comment:
I believe this was not implemented on purpose. @jcape probably knows why. Maybe we can get away by using the constant time eq here, or maybe there is a reason to just not allow this (which would mean manually `Eq` implementations for structs that need that and contain a `RistrettoPrivate`)

Reply:
After discussion with @jcape, we concluded that an implementation of Eq that enforces constant time comparison is acceptable.